examples/resources/hpegl_vmaas_load_balancer_virtual_server/nsx_t_lb_virtual_server.tf

```diff
@@ -2,12 +2,11 @@
 
 
 resource "hpegl_vmaas_load_balancer_virtual_server" "tf_lb_virtual_server" {
-  vip_name  =  "virtual"       
+  lb_id = data.hpegl_vmaas_load_balancer.lb.id 
```

Comment:
Same comment for the this one as well:
![image](https://user-images.githubusercontent.com/78580773/177924019-dedd375a-23cd-42fd-951b-650398f54c8d.png)